src/api/index.js

```diff
@@ -0,0 +1,5 @@
+import { request } from './api';
+
+export const getPeople = () => {
+  return request('');
```

Comment:
```suggestion
  return request();
```
if you don't use any params - remove them directly from request function declaration

Reply:
done